cocos/2d/CCNode.cpp

```diff
@@ -56,7 +56,7 @@ THE SOFTWARE.
 NS_CC_BEGIN
 
 // FIXME:: Yes, nodes might have a sort problem once every 30 days if the game runs at 60 FPS and each frame sprites are reordered.
-unsigned int Node::s_globalOrderOfArrival = 0;
+std::uint32_t Node::s_globalOrderOfArrival = 0;
```

Comment:
Is `std::` needed?

Reply:
include<cstdint> then use std::int32_t std::int64_t
include<stdint.h> then use int32_t int64_t